requirements-dev.txt

```diff
@@ -1,16 +1,12 @@
--r requirements.txt
```

Comment:
We have multiple requirements files in the requirements directory. Here I am trying to consolidate everything into only two: `requirements.txt` and `requirements-dev.txt` but it may take a few iterations until we get it right.